rockets-list.rb

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env ruby
+#
+# Affiche la liste formattée des messages Slack contenant `:rocket:`
+# Usage:
+#   SLACK_API_TOKEN=<token> ./rockets-to-text.rb
+
+require 'json'
+require 'net/http'
+require 'uri'
+
+if ENV['SLACK_API_TOKEN'] == nil
```

Comment:
Ça serait plus lisible, effectivement.

Seul pépin : `Object#present?` n'est pas une méthode de base de Ruby, c'est tiré de Rails (enfin, d'[ActiveSupport](http://guides.rubyonrails.org/active_support_core_extensions.html#blank-questionmark-and-present-questionmark)). Du coup ça nécessiterait de require une gem non standard dans ce script, donc de faire un Gemfile, donc… Bref, j'ai préféré garder le moins de dépendances possible :)

Tu vois une façon plus lisible d'écrire ça qui ne nécessiterait pas de gems externes ?